terraform/cmc.tf

```diff
@@ -0,0 +1,19 @@
+locals {
+  cm_non_prod_domains = ["mail-money-claims-nonprod.platform.hmcts.net"]
+  cm_prod_domains     = ["mail-money-claims-prod.platform.hmcts.net"]
```

Comment:
this domain uses service.gov.uk not platform.hmcts.net
```suggestion
  cmc_prod_domains     = ["mail.moneyclaims.service.gov.uk"]
```

Reply:
so - for production, we do not have any accounts and for the non-prod the above is correct as I have taken  them from the sendgrid,  see below screenshots
![image](https://user-images.githubusercontent.com/111435448/221547473-43f2e617-a895-41cc-b101-fd2bd1b17b9d.png)
![image](https://user-images.githubusercontent.com/111435448/221547540-4c439df4-55f4-4e6c-9810-f91ca1823038.png)

Reply:
see https://github.com/hmcts/sendgrid/pull/32#discussion_r1118603203